pkgs/by-name/bs/bsdgames/package.nix

```diff
@@ -27,13 +28,44 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
-    # Remove UTMPX support on Makefrag file
-    (fetchurl {
+    # Follow All patches from http://t2sde.org/packages/bsd-games.html
+    # May be removed in the next version
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/delay_output-sym.patch";
+      hash = "sha256-qbtp2cJVmEuxbTgpXM2gRHSNLE25OCmNxK+aeBBPRBw=";
+    })
+    (fetchpatch {
       url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/dm-noutmpx.patch";
-      sha256 = "1k3qp3jj0dksjr4dnppv6dvkwslrgk9c7p2n9vipqildpxgqp7w2";
+      hash = "sha256-oioJ5M7DAUVzX8k998p2h/APn8azw9Z8txmBuly2ouw=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/group-games.patch";
+      hash = "sha256-eG4H9VlCt8Js6XFcZ1tg8qsBoNcmM1dDAuRxEGY7PSk=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/hotfix-gcc43.patch";
+      hash = "sha256-9pkJmEOmM5vd/5nm3AlsbJNX0oX1kCtFrmciGpStvlA=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/hotfix-glibc.patch";
+      hash = "sha256-yNL88XORTXl2qSYTlYCYDN2G++TzBxywBpYdK+ufwrA=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/hotfix.patch";
+      hash = "sha256-UFXvtfVzJLRWvzbv0gaoVdt0vZOUj4UWWqoCZL/BDqE=";
+    })
+    (fetchpatch {
+      url = "http://svn.exactcode.de/t2/trunk/package/games/bsd-games/postfix-bsd.patch";
+      hash = "sha256-Noqq+FpSJb3heejGGHyLYYkHB3fC4qCv/p1XMfEGdx8=";
     })
   ];
 
+  env = {
+    NIX_CFLAGS_COMPILE = toString [
+      "-Wno-error=implicit-function-declaration"
```

Comment:
Arch simply drops the `sigpause` call which seems to work fine and cleaner (given the context of the call).

(This is the only change absolutely necessary to fix the build)